romanisim/parameters.py

```diff
@@ -4,6 +4,9 @@
 import numpy as np
 from astropy.time import Time
 from astropy import units as u
+import webbpsf as wpsf
+
+pixel_scale = wpsf.WFI().pixelscale
```

Comment:
I think Zach is saying that if you replace these two lines with pixel_scale = 0.11 then your build issues will go away.  That sounds fine.

Reply:
Thanks for your help, folks. Changed.